src/TConfig.cpp

```diff
@@ -62,7 +62,9 @@ static constexpr std::string_view StrPassword = "Password";
 
 // Misc
 static constexpr std::string_view StrHideUpdateMessages = "ImScaredOfUpdates";
+static constexpr std::string_view EnvStrHideUpdateMessages = "BEAMMP_HIDE_UPDATE_MESSAGES";
```

Comment:
one small consistency change
```suggestion
static constexpr std::string_view EnvStrHideUpdateMessages = "BEAMMP_IM_SCARED_OF_UPDATES";
```